### ISSUE TITLE
[MU4 Issue] Scaling menu too high

### ISSUE THREAD

**Issue by njvdberg** [musescore/MuseScore#8243]
**Describe the bug**
When changing the scaling, the menu appear far too high.

**To Reproduce**
1. Open a score
2. Click in the small arrow of the page scaling to change that scaling. The menu appears too high.

**Expected behavior**
The menu is expected to be just above the arrow, just like the `Page view` menu.

**Screenshots**
![Screenshot_20210603_200046](https://user-images.githubusercontent.com/48352523/120691481-d27db100-c4a6-11eb-949f-1eb293d2c3e8.png)
The menu starts roughly at the halve of the main window.

**Desktop (please complete the following information):**
 - OS: openSUSE Tumbleweed, KDE 5.21.5, KDE Frameworks 5.82.0

**Additional context**
none

**Comment by Eism** [comment]
@njvdberg Please explain how you changed the scaling?

**Comment by njvdberg** [comment]
@Eism Just click on the arrow (in the screenshot it is still blue) in the left lower corner of the main window.
To make sure, the **scaling** itself is correct, just the **position** of the scaling menu is wrong.
Strangely enough, the position of the `Page view` menu is correct, just above the field where I expect it.

I didn't try this on other platform (for the simple reason I have no access to other platforms).

**Comment by Eism** [comment]
@njvdberg What is the scaling value in your system?
We recently fixed a bug with scaling in popups :)

**Comment by njvdberg** [comment]
@Eism Not sure what you mean with `scaling value in your system` :(
But the scaling itself is correct, it is just the popup menu is too high.

I just found out the phenomena depends on the actual position of popup on the screen. If the popup appears on a screen Y coordinate of < 1024 the position of the popup is okay, it appears just above the control showing the actual scaling percentage.
However, if I move the lower edge of the main window down below 1024, the popup position is suddenly pushed up.
BTW, I just tested this with the latest master.
